ibis/expr/api.py

```diff
@@ -648,6 +648,33 @@ def random() -> ir.FloatingScalar:
 
     Similar to [](`random.random`) in the Python standard library.
 
+    ::: {.callout-note}
```

Comment:
Do you want to add a title for this note? This renders a bit nicer in the docs IMO. See e.g. https://ibis-project.org/reference/expression-tables#ibis.expr.types.relations.Table.sample.